master/reference/calicoctl/resources/bgppeer.md

```diff
@@ -2,15 +2,15 @@
 title: BGP Peer resource (bgpPeer)
 ---
 
-A BGP Peer resource represents a BGP peer which node(s) in this cluster will connect to. Configuration of BGP peers is required when configuring Calico to peer with your existing datacenter infrastructure (e.g. ToR). For more information on cluster layouts, see Calico's documentation on [L3 Topologies]({{site.baseurl}}/{{page.version}}/reference/private-cloud/l3-interconnect-fabric).
+A BGPPeer resource represents a BGP peer which node(s) in this cluster will connect to. Configuration of BGP peers is required when configuring Calico to peer with your existing datacenter infrastructure (e.g. ToR). For more information on cluster layouts, see Calico's documentation on [L3 Topologies]({{site.baseurl}}/{{page.version}}/reference/private-cloud/l3-interconnect-fabric).
```

Comment:
I think BGPPeer should be BGP Peer but perhaps with (bgpPeer) in parens after it.  It doesn't make sense to call it BGPPeer since the capitalization is actually different - so I'd rather make it more grammatical and then make it clear what the actual resource kind is.

I'm not 100% sure though.  WDYT?